app/services/decisions/dwp_decision_tree.rb

```diff
@@ -46,19 +49,37 @@ def after_confirm_details
     end
 
     def after_benefit_type
+      return edit(:partner_benefit_type) if partner_benefit_type_required?
+
+      benefit_check_routing(applicant)
+    end
+
+    def after_partner_benefit_type
+      benefit_check_routing(partner)
+    end
+
+    def benefit_check_routing(person)
       if form_object.benefit_type.none?
         if FeatureFlags.means_journey.enabled?
```

Comment:
I'd really like us to remove this Means feature flag. I certainly wouldn't want to trust that turning it off would work. We would have to do a massive amount of regression testing, etc.

Reply:
Me too!